### ISSUE TITLE
Adjust content on 'Email resent' page

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/notification-api#967]
Link text is unclear with “Not received an email” should be a question or call to action in English, be more descriptive of the outcome.

Heading is read by immersive screen reader as resent (resentment) rather than re-sent.

https://notification.alpha.canada.ca/email-not-received

https://notification.alpha.canada.ca/two-factor-email-sent?email_resent=True

![Screen Shot 2020-06-17 at 10 02 38 AM](https://user-images.githubusercontent.com/38330843/84907964-db048e00-b081-11ea-83dd-9a92b87bea0d.png)

**Comment by anikbrazeau** [comment]
https://github.com/cds-snc/notification-admin/pull/634
